### PR TITLE
add report table

### PR DESCRIPTION
**Pull request by wenzowski**
depends on #11

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ test:
 	@@$(MAKE) -s $(MAKEFLAGS) deploy;
 	@@$(MAKE) -s $(MAKEFLAGS) revert;
 	@@$(MAKE) -s $(MAKEFLAGS) deploy;
-	@@$(MAKE) -s $(MAKEFLAGS) prove;
+	@@$(MAKE) -s $(MAKEFLAGS) prove_unit;
```
Comment (wenzowski) [inline Makefile]:
```suggestion
	@@$(MAKE) -s $(MAKEFLAGS) prove_unit;
	@@$(MAKE) -s $(MAKEFLAGS) prove_style;
```

File: Makefile [hunk 2]
```diff
@@ -26,9 +26,14 @@ deploy:
...
-	# Run test suite using pg_prove
-	@@${PG_PROVE} -v -d ${TEST_DB} test/*
+prove_style:
+	# Run style-related test suite on all objects in db using pg_prove
+	@@${PG_PROVE} -v -d ${TEST_DB} test/**/*_test.sql
+.PHONY: test
+
+prove_unit:
+	# Run unit test suite using pg_prove
+	@@${PG_PROVE} -v -d ${TEST_DB} test/*_test.sql
```
Comment (wenzowski) [inline Makefile]: If we're splitting this out, let's get the unit tests in a `unit` folder.

File: Makefile [hunk 3]
```diff
@@ -26,9 +26,14 @@ deploy:
 	@@sqitch deploy ${TEST_DB};
 .PHONY: deploy
 
-prove:
-	# Run test suite using pg_prove
-	@@${PG_PROVE} -v -d ${TEST_DB} test/*
+prove_style:
+	# Run style-related test suite on all objects in db using pg_prove
+	@@${PG_PROVE} -v -d ${TEST_DB} test/**/*_test.sql
```
Comment (wenzowski) [inline Makefile]: as is, this will run all `*_test.sql` in all subfolders of `test/` due to the `/**/*` path globbing

File: Readme.md [hunk 4]
```diff
@@ -6,13 +6,21 @@ GGIRCS
 > for the Climate Action Secretariat
 
 ##Getting Started
```
Comment (wenzowski) [inline Readme.md]: If you're adjusting documentation, there should be a space between the `##` and the `Getting` in Markdown

File: deploy/table_raw_report.sql [hunk 5]
```diff
@@ -0,0 +1,16 @@
+-- Deploy ggircs:table_raw_report to pg
+-- requires: schema_ggircs_swrs
+
+begin;
+
+create table ggircs_swrs.raw_report (
```
Comment (hamzajaved) [inline deploy/table_raw_report.sql]: Update table name to ghgr_import

Comment (wenzowski) [reply deploy/table_raw_report.sql]: fixed during pairing

**Pull request merged**